### PR TITLE
Add Capabilities page type

### PR DESCRIPTION
**Pull request by Trombach**
## Description

This PR will add templates and wiring to enable the new capabilites page type in the ResearchHub. [Mock-ups](https://uoa.sharepoint.com/:b:/r/sites/ResearcherSkillsDevelopmentWorkingGroup/Shared%20Documents/General/24_ResearchHub/Final%20approach%20RSD%20mock%20up%20.pdf?csf=1&web=1&e=ckzcNb)

## Solution

- new module and page template for capabilities page and capabilities page list
- new graphQL queries
- update search lambda with new page type
- enable new page type in standard-card component

## Testing
- added unit tests for components (identical to article tests)
- added capability page fixture and e2e tests (identical to article tests)

## Have the changes been checked in the following browsers?
- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### CODE REVIEW

**Review by Trombach — COMMENTED** [review]
LGTM

**Review by devslashnoel — COMMENTED** [review]
Great work :100:,  just wondering about the min-height!

File: research-hub-web/src/app/components/cards/standard-card/standard-card.component.scss [hunk 1]
```diff
@@ -33,5 +33,6 @@ mat-icon {
 
 img {
   object-fit: cover;
+  min-height: 200px;
```
[the remark this answers is not in the document]
Comment (Trombach) [reply research-hub-web/src/app/components/cards/standard-card/standard-card.component.scss:36]: Added comment to explain reason

**Review by devslashnoel — APPROVED** [review]
Good job, looks good to me! :+1:

**Pull request merged**